content/v2.0-preview/get-started/get-started-with-managed-resources.md

```diff
@@ -16,10 +15,37 @@ This quickstart requires:
 * An AWS account with permissions to create an S3 storage bucket
 * AWS [access keys](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-creds)
 
+## About Managed Resources in Crossplane v2
+A _managed resource_ is anything Crossplane creates and manages outside of the
+Kubernetes cluster.
```

Comment:
Should we say control plane instead of cluster?

I go back and forth. "Kubernetes cluster" is more common and likely to make more sense to most people.

On the other hand "control plane" fits more closely with our framing of what Crossplane is.